cypress/integration/carry_okay_spec.js

```diff
@@ -1,89 +1,102 @@
 describe('CarryOkay', () => {
 
+  describe('Server error message', () => {
+  
+    it('Should display a message to the user if there and issue with the server', () =>{
+      cy.intercept({
+        method: 'GET', 
+        url:'/songs'
+      },
+      {
+        statusCode: 500,
+      })
+      cy.visit('http://localhost:3000');
+      cy.get('[data-cy=song-book-nav]').click();
+      cy.get('[data-cy=server-error-msg]').contains('Oops, our system seems to be down... how embarassing. Try again in a few moments!');
+  
+    });
+  });
+  
  beforeEach(() => {
-  cy.intercept( 'https://carryokay-server.herokuapp.com/genres', {fixture:"genre_data.json"})
-  cy.intercept('https://carryokay-server.herokuapp.com/songs', {fixture:"song_data.json"})
+  cy.intercept( '/genres', {fixture:"genre_data.json"})
+  cy.intercept('/songs', {fixture:"song_data.json"})
   cy.visit('http://localhost:3000');
  });
 
   describe('App Load', () => {
 
     it('should display App title and greeting', () => {
-      cy.get('h1').contains("CarryOkay");
-      cy.get('p').contains("Hello friend 😬");
+      cy.get('[data-cy=App-title]').contains("CarryOkay");
+      cy.get('[data-cy=home-greeting]').contains("Ready to sing?");
     });
 
     it('should contain Navigation links', () => {
-      cy.get('[data-cy=my-songs-nav]').contains("My Songs");
-      cy.get('[data-cy=song-book-nav]').contains("Song Book");
+      cy.get('[data-cy=my-songs-nav]').contains('My Songs');
+      cy.get('[data-cy=song-book-nav]').contains('Song Book');
     });
-
   });
 
   describe('Navigation', () => {
 
     it('should be able to navigate to My Songs view from Home view', () => {
       cy.get('[data-cy=my-songs-nav]').click();
-      cy.get("CarryOkay").should('not.exist');
-      cy.get("Hello friend 😬").should('not.exist');
-      cy.url().should("eq", "http://localhost:3000/mysongs")
```

Comment:
I'm assuming this means we are sticking with single quotes for the testing file, let me know if this is how we should be approaching the js files as well

Reply:
I noticed most of it was single quotes, so I went with that standard, in the .js files we might wanna decide on that. I think it's single quotes but we use double quotes in the JSX like how you use double quotes in HTML?